src/smexperiments/search_expression.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Simplify Search Expression by provide a simplified DSL"""
+from smexperiments._base_types import ApiObject
+from enum import Enum, unique
+
+
+@unique
+class Operator(Enum):
+    EQUALS = "Equals"
+    NOT_EQUALS = "NotEquals"
+    GREATER_THAN = "GreaterThan"
+    GREATER_THAN_OR_EQUAL = "GreaterThanOrEqualTo"
+    LESS_THAN = "LessThan"
+    LESS_THAN_OR_EQUAL = "LessThanOrEqualTo"
+    CONTAINS = "Contains"
```

Comment:
should the "In" operator also be in this list?

Reply:
Operator list is taken from boto spec: https://boto3.amazonaws.com/v1/documentation/api/1.12.8/reference/services/sagemaker.html#SageMaker.Client.search

I only listed all available options from there.

Reply:
IN will be released today.